recipes/repdenovo/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set version = "0.0.1" %}
+{% set sha256 = "95edb63bb1aa7ba55177f1c96ec534a12b040f640301d6e55fa286c7a9ea009b" %}
+{% set commit = "266495223e7176fdf67720454892dda6c4b8c35b" %}
+
+package:
+  name: repdenovo 
+  version: '{{version}}'
+
+source:
+  url: https://github.com/Reedwarbler/REPdenovo/archive/{{ commit }}.zip
+  sha256: {{ sha256 }}
+  patches:
+    - TERefiner.Makefile.patch
+    - ContigsMerger.Makefile.patch
+
+build:
+  number: 0
+  skip: true  # [osx]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - python=2.7
```

Comment:
Does it need this for installation? If so we should modify the `skip: true  # [osx]` to `skip: true  # [osx or py3k]`